packages/ukraine/README.md

```diff
@@ -0,0 +1,52 @@
+# 🇺🇦 Ukraine
+
+Contains all the cities and administrative centers of Ukraine:
+* Crimea
+* Cherkasy
+* Chernihiv
+* Chernivtsi
+* Dnipropetrovs'k
+* Donets'k
+* Ivano-Frankivs'k
+* Kharkiv
+* Kherson
+* Kiev City
+* Kiev
+* Kirovohrad
+* Luhans'k
+* L'viv
+* Mykolayiv
+* Odessa
+* Poltava
+* Rivne
+* Sevastpol' City
+* Sumy
+* Ternopil'
+* Transcarpathia
+* Vinnytsya
+* Volyn
+* Zaporizhzhya
+* Zhytomyr
```

Comment:
Could you use the English names (in the SVG file too) please?

Reply:
What do you mean? That's the original names
What i can do, i can remove apostrophes in the cities' names, e.g. "L'viv" to "Lviv"

Reply:
I looked on [wikipedia](https://en.wikipedia.org/wiki/Administrative_divisions_of_Ukraine#Regions) and the English names are a bit different (Dnipropetrovsk instead of Dnipropetrovs'k,  Zaporizhia instead of Zaporizhzhya...). However I don't know if it's a big deal, because these original names are still readable by most of people (unlike Cyrillic, Mandarin...) 🤔

If you know a better reference for the English names, feel free to share it!